Day7-Assignment/src/Components/AboutUs.js

```diff
@@ -0,0 +1,9 @@
+const AboutUs =()=>{
+    return(
+        <div class="about-us">
+        <h1>Welcome to our Team</h1>
+        <p>We are team of 10 members working in different domains, and We are here to learn React from scratch.</p>
+        </div>
+    )
```

Comment:
Try to proper indent html whenever possible.

Reply:
fixed